lib/workers/pr/index.ts

```diff
@@ -29,6 +30,10 @@ function noLeadingAtSymbol(input: string): string {
   return input.length && input.startsWith('@') ? input.slice(1) : input;
 }
 
+function nonEmptyStringAndNotWhitespace(input: string): boolean {
```

Comment:
Sorry, but please move it to `lib/util/string`. Seems useful for other places to me.

Reply:
Just did. Hard to say. `is` library seems the right place for it https://github.com/sindresorhus/is/issues/158

Reply:
Sure, but we don't know when it will be available  🤷‍♂️🤗